README.md

```diff
@@ -10,6 +10,7 @@ if you use HTTPS
 ```
 cd motohelper/
 gem install bundler
+brew install imagemagick (on OSX)
```

Comment:
Add this info please:
sudo apt-get install ruby-mini-magick (on  Ubuntu)

Reply:
Thanks for tip!